lib/node-utils.ts

```diff
@@ -50,6 +50,22 @@ export function isVariableDeclarator(
   return node && node.type === AST_NODE_TYPES.VariableDeclarator;
 }
 
+export function isRenderFunction(
```

Comment:
I thought this one was already moved to `node-utils.ts` but that was done in v4 of course 😓